package.json

```diff
@@ -32,7 +32,9 @@
     "wait-on": "wait-on",
     "workspace-schematic": "nx workspace-schematic",
     "preinstall": "npx npm-force-resolutions",
-    "gen-graphql": "graphql-codegen --config codegen.yml"
+    "gen-graphql": "graphql-codegen --config codegen.yml",
+    "seed:config": "ts-node --project ./apps/mull-api/tsconfig.app.json -r tsconfig-paths/register ./node_modules/typeorm-seeding/dist/cli.js config",
+    "seed:run": "ts-node --project ./apps/mull-api/tsconfig.app.json -r tsconfig-paths/register ./node_modules/typeorm-seeding/dist/cli.js seed"
```

Comment:
Oh Ritchelle reminded me, please add information about how these generator scripts in the readme